start/src/main/java/org/apache/accumulo/start/classloader/AccumuloClassLoader.java

```diff
@@ -87,7 +89,7 @@ public static String getAccumuloProperty(String propertyName, String defaultValu
     }
     try {
       var config = new PropertiesConfiguration();
-      try (var reader = new FileReader(accumuloConfigUrl.getFile())) {
+      try (var reader = new InputStreamReader(accumuloConfigUrl.openStream(), UTF_8)) {
```

Comment:
Looks like this changed got dinged by Sec bugs.  Should triage this before committing.

Reply:
Here is the reference, I don't think it applies: https://find-sec-bugs.github.io/bugs.htm#URLCONNECTION_SSRF_FD

Reply:
Yeah, I'm fixing that now. Also, my test didn't pass, so I still need to figure out if this actually fixes the bug I was seeing in the tests.